packages/core/src/toolkit-provider/ToolkitProvider.tsx

```diff
@@ -115,13 +115,13 @@ const getThemeName = (
   }
 };
 
-export const ToolkitProvider: FC<toolkitProvider> = ({
```

Comment:
Just curious why we're replacing arrow function here?

Reply:
More for consistency reasons. In other components, we use an arrow function because of forwardRef. Having a named function is beneficial for debugging too, since it automatically has a display name. This is minor since I think the named export helps there.